docs/integrations/webhooks/ilert.md

```diff
@@ -63,3 +113,17 @@ Follow the steps to configure the iLert Webhook.
 - For detailed information about webhook creation, refer to the [iLert Documentation](https://docs.ilert.com/integrations/webhook).
 - For support, [contact iLert](https://docs.ilert.com/contact).
 :::
+
+### Installing the iLert app
+
+{@import ../../reuse/apps/app-install-v2.md}
+
+## Viewing iLert dashboard
```

Comment:
I'd keep this plural (even though there's only one dashboard) for consistency with the rest of the corresponding app docs. Kind of like these headers, where they're plural no matter what.
<img width="605" alt="Screenshot 2023-12-22 at 5 16 40 AM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/4a4dab49-e70e-44ea-a81b-376a67857a9f">


```suggestion
## Viewing iLert dashboards
```